op-supernode/supernode/activity/superroot/superroot.go

```diff
@@ -44,37 +44,42 @@ type L2WithRequiredL1 struct {
 	MinRequiredL1 eth.BlockID
 }
 
-// atTimestampResponse is the response superroot_atTimestamp
+// AtTimestampResponse is the response superroot_atTimestamp
 // it contains:
 // - CurrentL1Derived: the current L1 block that each chain has derived up to (without any verification)
 // - CurrentL1Verified: the current L1 block that each verifier has processed up to
 // - VerifiedAtTimestamp: the L2 blocks which are fully verified at the given timestamp, and the minimum L1 block at which verification is possible
 // - OptimisticAtTimestamp: the L2 blocks which would be applied if verification were assumed to be successful, and their L1 sources
 // - SuperRoot: the superroot at the given timestamp using verified L2 blocks
-type atTimestampResponse struct {
+type AtTimestampResponse struct {
+	// TODO: We should probably specify json names for these
 	CurrentL1Derived      map[eth.ChainID]eth.BlockID
 	CurrentL1Verified     map[string]eth.BlockID
 	VerifiedAtTimestamp   map[eth.ChainID]L2WithRequiredL1
 	OptimisticAtTimestamp map[eth.ChainID]OutputWithSource
 	MinCurrentL1          eth.BlockID
-	MinVerifiedRequiredL1 eth.BlockID
-	SuperRoot             eth.Bytes32
+	MaxVerifiedRequiredL1 eth.BlockID
```

Comment:
Switched this to max instead of min because we want to know if all data required to fully verify the L2 blocks at the requested timestamp is available from the game's L1 head.